js/driver/virtio/index.js

```diff
@@ -35,3 +41,4 @@ function testDeviceId(deviceId) {
 }
 
 runtime.pci.addDriver(0x1af4, testDeviceId, driver);
+runtime.pci.addDriver(0x1af4, testDeviceId, driver);
```

Comment:
It would only read the subsystem value once, had to add it twice in order to read the subsystem id again and get the RNG driver.